### ISSUE TITLE
Upload Button Not Accessible via Keyboard Interaction

### ISSUE THREAD

**Issue by derekjackson-das** [danny-avila/LibreChat#4087]
**Issue Description**:
The "File Upload" button is not accessible to the keyboard. Currently the icon receives keyboard focus but pushing the space or enter key does not open the file menu as a mouse click does. Expected behavior is that when the upload file icon receives keyboard focus the user can hit the space or enter key to open the file menu and select a file to upload. 

**WCAG Criteria:**
[2.1.1 Keyboard - A](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html)

Screenshot or Screen Recording:
[File Upload Button Screen Recording](https://share.zight.com/xQuJY9wK)

Code Snippet:

```
<div class="flex" style="cursor: pointer;">
  <div id="audio-recorder" aria-label="Attach Files" 
       class="btn relative text-black focus:outline-none 
       focus:ring-2 focus:ring-border-xheavy 
       focus:ring-opacity-50 dark:text-white" 
       tabindex="0" style="padding: 0px;">
    <div class="flex w-full items-center justify-center gap-2">
      <svg width="24" height="24" viewBox="0 0 24 24" 
           fill="none" xmlns="http://www.w3.org/2000/svg" class="">

...

 </div> </div>
    <input multiple="" type="file" style="display: none;">
```

**Note**:
While the icon receives focus, the input element does not, meaning pressing the space bar or enter key does not activate the control. The `<input type="file">` element is also hidden using the `display:none` CSS style, which makes it programmatically hidden from assistive technologies and keyboard users.
This lack of accessibility impacts users relying on keyboards or assistive technologies to interact with file uploads.
The input element needs to be available for the keyboard (i.e. in the focus order) and assistive technology to interact with it. One option is to use a visually hidden CSS style that does not remove the element like display:none does. In the screen recording above there is an example making the input element visually hidden with the CSS pseudo class `:focus-within` applied on the parent container to trigger focus on the container block that has the SVG... this is one way to address this issue.

**Comment by danny-avila** [comment]
Clsoed by https://github.com/danny-avila/LibreChat/pull/4536
